cylc/flow/pathutil.py

```diff
@@ -17,16 +17,25 @@
 
 import errno
 import os
-from pathlib import Path
 import re
+from pathlib import Path
 from shutil import rmtree
```

Comment:
This diff goes against project style (sorts **p**athlib after **r**e), but there's a config to change this if we would like to stick with the status quo.